etc/adminhtml/system/adyen_oneclick.xml

```diff
@@ -24,7 +24,7 @@
 -->
 <include xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_include.xsd">
     <group id="adyen_oneclick" translate="label" type="text" sortOrder="150" showInDefault="1" showInWebsite="1" showInStore="1">
-        <label><![CDATA[Stored Payment Methods - Requires Adyen Credit Card]]></label>
+        <label><![CDATA[Stored Payment Methods]]></label>
```

Comment:
@AlexandrosMor do you know why this was removed? I moved the adyen_hpp_vault active flag to adyen_hpp.xml so maybe this doesn't apply anymore

Reply:
originally we discussed to put the config into the stored payment method section because it's more related to stored payments than hpp. Of course it's just a matter of documenting it

Reply:
Right. I could move it back there but I changed the config path so Magento picks up the active flag natively, and the new path seems to fit better in adyen_hpp

Reply:
Ok, keep it where it is and then you can add back the removed text from the label

Reply:
```suggestion
        <label><![CDATA[Stored Payment Methods - Requires Adyen Credit Card]]></label>
```